tests/test_nifgen.py

```diff
@@ -35,7 +35,7 @@ def test_get_all_nifgen_instrument_names(self, standalone_tsm_context):
     def test_set_nifgen_session(self, standalone_tsm_context):
         instrument_names = standalone_tsm_context.get_all_nifgen_instrument_names()
         for instrument_name in instrument_names:
-            with nifgen.Session("", options=OPTIONS) as session:
+            with nifgen.Session(instrument_name, options=OPTIONS) as session:

```

Comment:
I don't understand why, but sometimes opening nifgen sessions with empty strings causes an error. This should fix intermittent errors in the pipeline.